games/o.yaml

```diff
@@ -13,6 +13,22 @@
   type: remake
   updated: 2016-02-08
 
+- name: OceanPop
+  images:
+  - https://raw.githubusercontent.com/sharkwouter/oceanpop/1.0/screenshot.jpg
+  langs:
+  - C++
+  licenses:
+  - MIT
+  development: sporadic
+  originals:
+  - HuniePop
```

Comment:
I think something like bejeweled would be better here; I don't see any similarity with huniepop specifically except for the name "pop"
wondering what @sharkwouter thinks

Reply:
Sure, I can eloborate. In Huniepop you have a specific amount of moves in which you have to reach a target score and you are able to move gems as far as you'd like. OceanPop implements both of those mechanics. This makes it quite different from Bejeweled.